### PR TITLE
[*] MO : New version

### PR DESCRIPTION
**Pull request by M03G**

### CODE REVIEW

File: views/templates/admin/options_certificate.tpl [hunk 1]
```diff
@@ -40,17 +40,17 @@
 	    
 	    <div class="form-group">
                 <label class="control-label col-lg-3">{l s='Language' mod='trustedshops'}</label>
-                <div class="col-lg-4">{$lang|escape:'html':'UTF-8'}</div>
+                <div class="col-lg-4"><output style="font-size: 13px;">{$lang|escape:'html':'UTF-8'}</output></div>
```
Comment (Quetzacoalt91) [inline views/templates/admin/options_certificate.tpl]: As you can see, you wrote `style="font-size: 13px;"` for all your output tags. You should move these style definition in a CSS file.

**Comment by Quetzacoalt91** [comment]
Hi @M03G,

A few warnings and errors reported by the [validator](https://validator.prestashop.com) could be easily fixed. Can you please have a look on it ?

Thanks

**Comment by M03G** [comment]
Hi @Quetzacoalt91 
I changed the module. Please take a look.

File: views/templates/front/order-confirmation-tsbp-excellence.tpl [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <span id="tsCheckoutAmount">{$buyer_protection.amount|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutCurrency">{$buyer_protection.currency|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutPaymentType">{$buyer_protection.paymentType|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutBuyerEmail">{$buyer_protection.buyerEmail|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutShopCustomerID">{$buyer_protection.shopCustomerID|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutShopOrderID">{$buyer_protection.shopOrderID|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutOrderDate">{$buyer_protection.orderDate|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutShopSystemVersion">{$buyer_protection.shopSystemVersion|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutWsUser">{$buyer_protection.wsUser|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutWsPassword">{$buyer_protection.wsPassword|escape:'html':'UTF-8'}</span>
```
Comment (Quetzacoalt91) [inline views/templates/front/order-confirmation-tsbp-excellence.tpl]: Heu, which user / password are you going to display ?

File: views/templates/front/order-confirmation-tsbp-excellence_1.5.tpl [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <span id="tsCheckoutAmount">{$buyer_protection.amount|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutCurrency">{$buyer_protection.currency|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutPaymentType">{$buyer_protection.paymentType|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutBuyerEmail">{$buyer_protection.buyerEmail|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutShopCustomerID">{$buyer_protection.shopCustomerID|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutShopOrderID">{$buyer_protection.shopOrderID|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutOrderDate">{$buyer_protection.orderDate|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutShopSystemVersion">{$buyer_protection.shopSystemVersion|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutWsUser">{$buyer_protection.wsUser|escape:'html':'UTF-8'}</span>
+        <span id="tsCheckoutWsPassword">{$buyer_protection.wsPassword|escape:'html':'UTF-8'}</span>
```
Comment (Quetzacoalt91) [inline views/templates/front/order-confirmation-tsbp-excellence_1.5.tpl]: Same question about the user / password

File: trustedshops.php [hunk 4]
```diff
@@ -40,21 +40,24 @@ class TrustedShops extends Module
...
 		$this->author = 'silbersaiten';
 		$this->bootstrap = true;
-		
-		self::$template_version = version_compare(_PS_VERSION_, '1.6', '<') ? '1.5' : '1.6';
 
+		self::$template_version = version_compare(_PS_VERSION_, '1.6', '<') ? '1.5' : '1.6';
+		$this->ps_versions_compliancy = array(
+			'min' => '1.5.0.0',
+			'max' => _PS_VERSION_
+		);
```
Comment (Quetzacoalt91) [inline trustedshops.php]: Unfortunately, we have a bug on specific versions of PrestaShop 1.5 which don't manage the ps_versions_compliancy properly. Can you please remove this variable ?

**Comment by Quetzacoalt91** [comment]
Hi @M03G,

Can you please update the file config.xml ? It also stores the module version.

**Comment by M03G** [comment]
Hi @Quetzacoalt91 
You're right. Fixed.

File: trustedshops.php (hunk 4 above)
Comment (M03G) [inline trustedshops.php]: Trustedshops asked to add this variable. You can otherwise specify the minimum version? Maybe change in 1.5.0.13?

Comment (Quetzacoalt91) [inline trustedshops.php]: We can specify it on PrestaShop addons, so it can't be available for merchants on PS 1.4 :)

Comment (igarba) [inline trustedshops.php]: Hi @M03G, we didn't ask to add the variable, just the min and max compatible versions, and that would be 1.5.x -  1.6.0.14

Comment (igarba) [inline trustedshops.php]: @Quetzacoalt91, the module is not compatible with versions below 1.5, or what do you mean?

Comment (Quetzacoalt91) [inline trustedshops.php]: I am saying that if you do not specify the versions compatible with this module, we can do it anyway on our side on PrestaShop Addons.

Comment (igarba) [inline trustedshops.php]: OK, I now understand ;)
@M03G, could you please replace the variable with the actual Prestashop version? Thanks a lot!

**Comment by Quetzacoalt91** [comment]
Hi @M03G & @igarba,

Thanks, the PR has just been merged and the changes should be available from tomorrow.

Regards

**Comment by igarba** [comment]
Nice :+1:  Thank you

**Pull request merged**